app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterDetailPresenter.kt

```diff
@@ -0,0 +1,28 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.mvp
+
+import com.puzzlebench.clean_marvel_kotlin.domain.usecase.GetCharacterDetailServiceUseCase
+import com.puzzlebench.clean_marvel_kotlin.presentation.base.Presenter
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+
+class CharacterDetailPresenter(view: CharacterDetailView, private val getChatacterDetailServiceUseCase: GetCharacterDetailServiceUseCase, val subscriptions: CompositeDisposable) : Presenter<CharacterDetailView>(view) {
+    fun init() {
+        view.init()
+        requestGetCharacterDetail()
```

Comment:
Please remove the call for the request method from the init and call it from activity/fragment